src/pybind/mgr/dashboard/module.py

```diff
@@ -405,6 +407,26 @@ def set_mgr_created_self_signed_cert(self):
         if result.retval != 0:
             return result
         return 0, 'Self-signed certificate created', ''
+    
+    @CLICommand("dashboard get issue")
+    def get_issues_cli(self, issue_number: str):
+        cephTrackerClient = CephTrackerClient()
+        response = cephTrackerClient.get_issues(issue_number)
+        return 0,str(response["issue"]), ''
```

Comment:
This command needs documentation and I would suggest taking care of a `None` `issue_number` since it creates an error.
```
[root@ceph ceph]# ceph dashboard get issue
2021-08-20T14:09:07.460+0000 7f36ff1c0700 -1 WARNING: all dangerous and experimental features are enabled.
2021-08-20T14:09:07.469+0000 7f36ff1c0700 -1 WARNING: all dangerous and experimental features are enabled.
Error EINVAL: Traceback (most recent call last):
  File "/ceph/src/pybind/mgr/mgr_module.py", line 1566, in _handle_command
    return CLICommand.COMMANDS[cmd['prefix']].call(self, cmd, inbuf)
  File "/ceph/src/pybind/mgr/mgr_module.py", line 415, in call
    return self.func(mgr, **kwargs)
  File "/ceph/src/pybind/mgr/dashboard/module.py", line 414, in get_issues_cli
    response = cephTrackerClient.get_issues(issue_number)
  File "/ceph/src/pybind/mgr/dashboard/rest_client.py", line 534, in func_wrapper
    **kwargs)
  File "/ceph/src/pybind/mgr/dashboard/services/feedback.py", line 49, in get_issues
    response = request()
  File "/ceph/src/pybind/mgr/dashboard/rest_client.py", line 323, in __call__
    data, raw_content, headers)
  File "/ceph/src/pybind/mgr/dashboard/rest_client.py", line 452, in do_request
    resp.content)
dashboard.rest_client.RequestException: CephTracker REST API failed request with status code 404
b''
```
Good job tho!

Reply:
Did error handling for missing issue number